src/buildADTrecursion.c

```diff
@@ -48,7 +46,8 @@ void buildADTrecursion(double *coord,double *adtReals,double *adtWork,int *adtIn
     // reorder elements with nleft elements to
     // the left of median of adtWork
     //
-    median_(elementsAvailable,adtWork,&nav,&coordmid);
+    std::sort(elementsAvailable, elementsAvailable + nav,
```

Comment:
Appears as though median_ is changing both elementsAvailable and adtWork ? But the new algorithm only changes elementsAvailable ?

Reply:
are you looking at the right commit? I don't see this diff in the final version

Reply:
Yeah it's weird it's showing the sort call here, but yeah let me look at this again. Yeah this has indexes as well, so I can sort both.

Reply:
Ok so the comparison needs to happen on `adtWork`, but `elementsAvailable` is what needs to be reordered. So `adtWork` is only used for the sort and not after. So I used `adtWork` for the comparison.